packages/api/src/server.ts

```diff
@@ -35,7 +35,8 @@ export class WepublishServer {
     })
 
     const corsOptions = {
-      origin: '*',
+      origin: true,
+      credentials: true,
```

Comment:
Setting `credentials: true` might be dangerous in combination with `origin: true` if we support cookie authorization in the future, that's one of the reason it isn't allowed in combination with `origin: "*"`. It shouldn't be needed at the moment anyway `Authorization` is in the `allowedHeaders`.

The only way to add `credentials: true` securely would be to allow the configuration of the `origin`.